### DISCUSSION TITLE
Weaviate integration

### DISCUSSION THREAD

**Discussion by Verizane** [Arize-ai/phoenix#1652] [Q&A] [answered]
Is it possible to easily integrate a weaviate embeddings vectorstore into embeddings analysis? And if so, how do I do it? I could not find anything in the documentation about that.

**Comment by mikeldking** [comment, marked as answer]
👋 @Verizane thanks for checking the project out! We actually don't integrate directly with vector stores but rather just support visualizing them along with your query embeddings. Example using line one can be found here [https://github.com/Arize-ai/phoenix/blob/main/tutorials/langchain_pinecone_search_and_retrieval_tutorial.ipynb](https://github.com/Arize-ai/phoenix/blob/main/tutorials/langchain_pinecone_search_and_retrieval_tutorial.ipynb)

You can find the details of uploading a corpus data set in our documentation [here](https://docs.arize.com/phoenix/how-to/define-your-schema/corpus-data)

Hope that helps! Definitely can get you in touch with the right people if you have any issues extracting data from weaviate
